### ISSUE TITLE
bug: openllm download llama meta-llama/Llama-2-70b-chat attempts to download 7b parameter variant

### ISSUE THREAD

**Issue by zacps** [bentoml/OpenLLM#180]
### Describe the bug

As title, `openllm download llama meta-llama/Llama-2-70b-chat` attempts to download 7b parameter variant.

### To reproduce

Run `openllm download llama meta-llama/Llama-2-70b-chat`

### Logs

```shell
'llama' with model_id=meta-llama/Llama-2-70b-chat does not exists in local store. Saving to BENTOML_HOME...
Loading checkpoint shards: 100%|██████████████████████████████████████████████████████████████████| 2/2 [00:04<00:00,  2.16s/it]
Using pad_token, but it is not set yet.
Saved model: pt-huggyllama-llama-7b:8416d3fefb0cb3ff5775a7b13c1692d10ff1aa16
```

### Environment

```bash
BENTOML_DEBUG=''
BENTOML_QUIET=''
BENTOML_BUNDLE_LOCAL_BUILD=''
BENTOML_DO_NOT_TRACK=''
BENTOML_CONFIG=''
BENTOML_CONFIG_OPTIONS=''
BENTOML_PORT=''
BENTOML_HOST=''
BENTOML_API_WORKERS=''
```

#### System information

`bentoml`: 1.1.1
`python`: 3.8.10
`platform`: Linux-5.4.0-155-generic-x86_64-with-glibc2.29
`uid_gid`: 1003:1003
<details><summary><code>pip_packages</code></summary>

<br>

```
absl-py==0.15.0
accelerate==0.21.0
aiohttp==3.8.4
aiosignal==1.3.1
anyio==3.6.1
appdirs==1.4.3
argon2-cffi==21.3.0
argon2-cffi-bindings==21.2.0
asgiref==3.7.2
astunparse==1.6.2
async-timeout==4.0.2
attrs==23.1.0
auto-gptq==0.3.1
Automat==0.8.0
Babel==2.10.3
backcall==0.1.0
beautifulsoup4==4.11.1
bentoml==1.1.1
bio==1.3.9
biopython==1.79
biothings-client==0.2.6
bitsandbytes==0.41.0
bleach==3.1.1
blinker==1.4
boto3==1.24.6
botocore==1.27.6
build==0.10.0
CacheControl==0.12.10
cachetools==4.0.0
cachy==0.3.0
caffe==1.0.0
cattrs==23.1.2
certifi==2019.11.28
cffi==1.15.1
chardet==3.0.4
charset-normalizer==3.1.0
circus==0.18.0
cleo==0.8.1
click==8.1.6
click-option-group==0.5.6
clikit==0.6.2
cloud-init==23.2.1
cloudpickle==2.2.1
colorama==0.4.3
coloredlogs==15.0.1
command-not-found==0.3
configobj==5.0.6
constantly==15.1.0
contextlib2==21.6.0
crashtest==0.3.1
cryptography==2.8
cuda-python==12.2.0
cycler==0.10.0
Cython==3.0.0
dataclasses-json==0.5.7
datasets==2.14.2
dbus-python==1.2.16
decorator==4.4.2
deepmerge==1.1.0
defusedxml==0.7.1
Deprecated==1.2.14
dill==0.3.7
distlib==0.3.4
distro==1.4.0
distro-info===0.23ubuntu1
docker-pycreds==0.4.0
entrypoints==0.3
exceptiongroup==1.1.2
fairscale==0.4.13
fastjsonschema==2.15.3
filelock==3.6.0
filetype==1.2.0
flake8==3.7.9
flatbuffers==1.12
frozenlist==1.3.3
fs==2.4.16
fsspec==2023.6.0
future==0.18.2
gast==0.4.0
gitdb==4.0.10
GitPython==3.1.31
google-auth==1.5.1
google-auth-oauthlib==0.4.1
google-pasta==0.2.0
greenlet==2.0.2
grpcio==1.56.2
grpcio-health-checking==1.56.2
h11==0.14.0
h5py==2.10.0
html5lib==1.0.1
httpcore==0.17.3
httplib2==0.14.0
httpx==0.24.1
huggingface-hub==0.16.4
humanfriendly==10.0
hyperlink==19.0.0
idna==2.8
imageio==2.4.1
importlib-metadata==6.0.1
incremental==16.10.1
inflection==0.5.1
ipykernel==5.2.0
ipython==7.13.0
ipython_genutils==0.2.0
jax==0.4.13
jaxlib==0.4.13
jedi==0.15.2
jeepney==0.7.1
Jinja2==3.1.2
jmespath==1.0.0
joblib==1.2.0
json5==0.9.8
jsonpatch==1.22
jsonpointer==2.0
jsonschema==3.2.0
jupyter-client==7.3.4
jupyter-console==6.0.0
jupyter-core==4.6.3
jupyter-server==1.18.1
jupyterlab==3.4.3
jupyterlab-pygments==0.2.2
jupyterlab-server==2.15.0
keras==2.13.1
Keras-Applications==1.0.8
Keras-Preprocessing==1.1.2
keyring==23.5.0
kiwisolver==1.0.1
langchain==0.0.178
language-selector==0.1
launchpadlib==1.10.13
lazr.restfulclient==0.14.2
lazr.uri==1.0.3
lockfile==0.12.2
Mako==1.1.0
Markdown==3.1.1
markdown-it-py==3.0.0
MarkupSafe==2.1.1
marshmallow==3.19.0
marshmallow-enum==1.5.1
matplotlib==3.1.2
mccabe==0.6.1
mdurl==0.1.2
mistune==0.8.4
ml-dtypes==0.2.0
modAL==0.4.1
more-itertools==4.2.0
mpi4py==3.0.3
mpmath==1.1.0
msgpack==1.0.3
multidict==6.0.4
multiprocess==0.70.15
mygene==3.2.2
mypy-extensions==1.0.0
nbclassic==0.4.3
nbclient==0.6.6
nbconvert==6.5.0
nbformat==5.4.0
nest-asyncio==1.5.5
netifaces==0.10.4
networkx==2.4
notebook-shim==0.1.0
numexpr==2.8.4
numpy==1.23.5
nvidia-ml-py==7.352.0
oauthlib==3.1.0
openapi-schema-pydantic==1.2.4
openllm==0.2.12
opentelemetry-api==1.18.0
opentelemetry-instrumentation==0.39b0
opentelemetry-instrumentation-aiohttp-client==0.39b0
opentelemetry-instrumentation-asgi==0.39b0
opentelemetry-instrumentation-grpc==0.39b0
opentelemetry-sdk==1.18.0
opentelemetry-semantic-conventions==0.39b0
opentelemetry-util-http==0.39b0
opt-einsum==3.3.0
optimum==1.10.1
orjson==3.9.2
packaging==23.1
pandas==1.5.3
pandocfilters==1.5.0
parso==0.5.2
pastel==0.2.1
pathspec==0.11.2
pathtools==0.1.2
pbr==5.8.1
peft==0.4.0
pexpect==4.8.0
pickleshare==0.7.5
Pillow==7.0.0
pip-requirements-parser==32.0.1
pip-tools==7.2.0
pkg_resources==0.0.0
pkginfo==1.8.2
platformdirs==2.5.1
poetry-core==1.0.8
prometheus-client==0.14.1
prompt-toolkit==2.0.10
protobuf==4.22.1
psutil==5.9.4
ptyprocess==0.7.0
pyarrow==12.0.1
pyasn1==0.4.2
pyasn1-modules==0.2.1
pybind11==2.10.1
pycodestyle==2.5.0
pycparser==2.21
pycuda==2019.1.2
pydantic==1.10.7
pyflakes==2.1.1
Pygments==2.15.1
PyGObject==3.36.0
pygpu==0.7.6
PyHamcrest==1.9.0
PyJWT==1.7.1
pylev==1.4.0
pymacaroons==0.13.0
PyNaCl==1.3.0
pynvml==11.5.0
pyOpenSSL==19.0.0
pyparsing==2.4.6
pyproject_hooks==1.0.0
pyrsistent==0.15.5
pyserial==3.4
python-apt==2.0.1+ubuntu0.20.4.1
python-dateutil==2.8.2
python-debian==0.1.36+ubuntu1.1
python-json-logger==2.0.7
python-multipart==0.0.6
pytools==2019.1.1
pytz==2023.3
PyWavelets==0.5.1
PyYAML==6.0
pyzmq==18.1.1
regex==2023.6.3
requests==2.22.0
requests-oauthlib==1.0.0
requests-toolbelt==0.9.1
requests-unixsocket==0.2.0
rich==13.5.2
rouge==1.0.1
rsa==4.0
s3transfer==0.6.0
safetensors==0.3.1
schema==0.7.5
scikit-cuda==0.5.3
scikit-image==0.16.2
scikit-learn==1.0.2
scipy==1.10.1
SecretStorage==3.3.1
Send2Trash==1.8.0
sentencepiece==0.1.99
sentry-sdk==1.17.0
service-identity==18.1.0
setproctitle==1.3.2
shellingham==1.4.0
simple-di==0.1.5
simplejson==3.16.0
six==1.16.0
smmap==5.0.0
sniffio==1.2.0
sos==4.4
soupsieve==2.3.2.post1
SQLAlchemy==2.0.15
ssh-import-id==5.10
starlette==0.28.0
sympy==1.5.1
systemd-python==234
tabulate==0.9.0
tenacity==8.2.2
tensorboard==2.13.0
tensorflow-estimator==2.13.0
tensorflow-gpu==2.13.0
termcolor==1.1.0
terminado==0.15.0
testresources==2.0.1
Theano==1.0.4
threadpoolctl==3.1.0
tinycss2==1.1.1
tokenizers==0.13.3
tomli==2.0.1
tomlkit==0.10.0
torch==2.0.1
torchvision==0.15.2
tornado==6.2
tqdm==4.64.0
traitlets==5.3.0
transformers==4.31.0
triton==2.0.0
Twisted==18.9.0
typing-inspect==0.8.0
typing_extensions==4.6.0
tzdata==2023.3
ubuntu-advantage-tools==8001
ufw==0.36
unattended-upgrades==0.1
urllib3==1.26.15
uvicorn==0.23.2
virtualenv==20.0.17
wadllib==1.3.3
wandb==0.14.0
watchfiles==0.19.0
wcwidth==0.1.8
webencodings==0.5.1
websocket-client==1.3.3
Werkzeug==0.16.1
wrapt==1.11.2
xxhash==3.3.0
yarl==1.9.2
zipp==1.0.0
zope.interface==4.7.1
```

</details>

**Comment by aarnphm** [comment]
Yes I have fixed this bug on 0.2.12

**Comment by aarnphm** [comment]
Hey there can you try out with 0.2.13? ths.
